platform/android/java/lib/src/org/godotengine/godot/Godot.kt

```diff
@@ -757,6 +761,8 @@ class Godot private constructor(val context: Context) {
 			darkMode = newDarkMode
 			GodotLib.onNightModeChanged()
 		}
+
+		GodotLib.onCameraRotationChange()
```

Comment:
Configuration change callback does not necessarily mean that there was a screen orientation change, so you should check specifically for it before calling `onCameraRotationChange()`.